es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super();
+    this.radius = radius;
+    this.area = radius * Math.PI;
+    this.circumference = 2 * radius * Math.PI;
```

Comment:
So, we will have to "override" Shape's constructor values (area and circumference) from Circle. How do we do that?

Reply:
I tried different ways, but I'm not sure which is the correct or most efficient way to do it.

Reply:
Looks great! :)